eng/Versions.props

```diff
@@ -20,8 +20,8 @@
     <MicrosoftDeveloperControlPlanewindowsarm64PackageVersion>0.1.50</MicrosoftDeveloperControlPlanewindowsarm64PackageVersion>
     <MicrosoftNETCoreAppRefPackageVersion>8.0.0-rc.1.23419.3</MicrosoftNETCoreAppRefPackageVersion>
     <MicrosoftmacOSSdkPackageVersion>13.3.8825-net8-rc1</MicrosoftmacOSSdkPackageVersion>
-    <MicrosoftDotNetRemoteExecutorPackageVersion>8.0.0-beta.24081.5</MicrosoftDotNetRemoteExecutorPackageVersion>
-    <MicrosoftDotNetXUnitExtensionsPackageVersion>8.0.0-beta.24081.5</MicrosoftDotNetXUnitExtensionsPackageVersion>
+    <MicrosoftDotNetRemoteExecutorPackageVersion>8.0.0-beta.24109.1</MicrosoftDotNetRemoteExecutorPackageVersion>
```

Comment:
do these need corresponding version xml file changes?

Reply:
Good catch, will fix.

Reply:
I ended up undoing. These were coming from arcade, but were package versions from builds that didn't pass validation, which is why we didn't get the update of those versions from arcade.